src/main/scala/fr/polytechnique/cmap/cnam/etl/events/HospitalStay.scala

```diff
@@ -27,4 +27,16 @@ trait HospitalStay extends AnyEvent with EventBuilder {
     apply(patientID, hospitalID, hospitalID, 0D, start, Some(end))
 }
 
-object HospitalStay extends HospitalStay
\ No newline at end of file
+object HospitalStay extends HospitalStay
+
+object McoHospitalStay extends HospitalStay {
+  override val category: EventCategory[HospitalStay] = "mco_hospital_stay"
+}
+
+object SsrHospitalStay extends HospitalStay {
+  override val category: EventCategory[HospitalStay] = "ssr_hospital_stay"
+}
+
+object HadHospitalStay extends HospitalStay {
```

Comment:
Do we have the concept of Hospital Stay? Isn't a HospitalStay at home?

Reply:
See above comments. When discussing with medical experts (Haute Autorité de Santé or Drees people), they refers to these events as hospitalStays.